package.json

```diff
@@ -3,15 +3,18 @@
   "version": "0.0.1",
   "license": "UNLICENSED",
   "scripts": {
-    "start": "npm run stop && concurrently \"npm run server\" \"npm run expose\"",
+    "expose": "ngrok http 8302",
+    "get-public-address": "addr=$(curl http://localhost:4040/api/tunnels | jq -r '.tunnels | .[] | select( .name == \"command_line\" ) | .public_url') && echo \"\nYour public URL is $addr\"",
```

Comment:
What is this `get-public-address` for?

Reply:
`npm start` starts ngrok, but it's an extra manual step to go to `http://localhost:4040/status` to retrieve your public ngrok address. With `get-public-address`, you can get the address in your terminal when you run `npm start` and just copy the URL easily.